<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GrblStream</class>
 <widget class="QMainWindow" name="GrblStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="circularProgressBarBase">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>460</width>
      <height>460</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circularProgress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>440</width>
       <height>440</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 220px;
	background-color: qconicalgradient(cx:0.5, cy:0.5, angle:90, stop:0.749 rgba(0, 170, 0, 0), stop:0.750 rgba(0, 85, 127, 255));
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circularBig">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>440</width>
       <height>440</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 220px;
	background-color: rgba(0, 240, 112, 120);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="container">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>400</width>
       <height>400</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	border-radius: 200px;
	background-color: rgb(0, 240, 112);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="send_label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>241</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: #FFFFFFFF</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;PRESS &lt;/span&gt;PLAY&lt;span style=&quot; font-weight:400;&quot;&gt; TO START&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="gcode_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: #FFFFFFFF</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="progress_label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>321</width>
        <height>181</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>100</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0);
color: #FFFFFFFF</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;span style=&quot; font-size:72pt; vertical-align:super;&quot;&gt;%&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="engrave_label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>270</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color: rgb(127, 240, 180);
color: #FFFFFFFF</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Ready to engrave&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="play_pause_button">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>310</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(0, 255, 119);
	color:rgb(255, 255, 255);
	border-radius: 35px;
}
QPushButton:hover{
	background-color:rgb(0, 240, 112);
}
QPushButton:pressed{
	background-color:rgb(0, 143, 66);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Images/PinClipart.com_barcode-clipart_2064177.png</normaloff>
        <normalon>Images/PinClipart.com_empty-refrigerator-clipart_1968920.png</normalon>Images/PinClipart.com_barcode-clipart_2064177.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>130</width>
        <height>200</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="stop_button">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>310</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Segoe UI Black</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color:rgb(0, 255, 119);
	color:rgb(255, 255, 255);
	border-radius: 35px;
}
QPushButton:hover{
	background-color:rgb(0, 240, 112);
}
QPushButton:pressed{
	background-color:rgb(0, 143, 66);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>Images/pngwing.com.png</normaloff>Images/pngwing.com.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>120</width>
        <height>200</height>
       </size>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <zorder>circularBig</zorder>
    <zorder>circularProgress</zorder>
    <zorder>container</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
